README.md

```diff
@@ -194,7 +207,7 @@ presto --server localhost:8080 --catalog hive --schema default --user presto --f
 
 ## License
 
-________
+---
 
 ## References
```

Comment:
Table of contents does not include links to section that are present after **License** section.
Also, needs reordering.

Reply:
I will move this section in example/resources/data dir